client/marklogicClient.ts

```diff
@@ -363,30 +363,27 @@ export async function filterJsServerByConnectedStatus(
 ): Promise<MlxprsQuickPickItem[]> {
     const requests = [];
     const filteredChoices: MlxprsQuickPickItem[] = [];
-    choices.forEach(async (choice) => {
-        const url = ClientContext.buildUrl(dbClientContext.params.host, `/jsdbg/v1/connected/${choice.label}`, dbClientContext.params.ssl, managePort);
-        const options = {
-            headers: {
-                'Content-type': 'application/x-www-form-urlencoded',
-                'X-Error-Accept': 'application/json'
-            },
-            auth: {
-                user: dbClientContext.params.user,
-                pass: dbClientContext.params.pwd,
-                'sendImmediately': false
-            }
-        };
-        if (dbClientContext.params.pathToCa !== '') {
-            options['agentOptions'] = { ca: fs.readFileSync(dbClientContext.params.pathToCa) };
-        }
-        options['rejectUnauthorized'] = dbClientContext.params.rejectUnauthorized;
-        const connectedRequest = request.get(url, options)
-            .then((response) => {
-                if (response === requiredResponse) {
+
+    const manageClientParams: MlClientParameters = JSON.parse(JSON.stringify(dbClientContext.params));
+    manageClientParams.port = managePort;
+    manageClientParams.contentDb = null;
+    manageClientParams.modulesDb = null;
```

Comment:
This is the same as `newMarkLogicManageClient` in `mlRuntime.ts` - perhaps that function should instead be in here so that it can be reused both here and in `mlRuntime.ts`?

Reply:
I thought about it. Decided it wasn't worth it. But based on your comment, I've gone ahead and refactored it.